scripts/hubot-netrunnerdb-cards.coffee

```diff
@@ -314,3 +344,31 @@ module.exports = (robot) ->
 				cardString += " + " + randomCard.title
 
 		res.send cardString
+
+	robot.hear /^!find (.*)/, (res) ->
+		conditions = []
+		for part in res.match[1].toLowerCase().split(" ")
+			if out = part.match(/([a-z]+)(:|=|<|>|!)([\w]+)/)
```

Comment:
Should the `([a-z]+)` group be limited only to valid filter parameters (`etsfxpondcaiuy`)? Also, does it need to allow multiple characters for a parameter? It appears that search filter keys are always single characters, so I don't think a `+` is necessary here, unless I'm missing something.

With that limitation, the conditionals below could probably be slightly simplified.